src/command/dev/next/router/run.rs

```diff
@@ -158,6 +163,7 @@ impl RunRouter<state::Run> {
             .call(
                 WriteFileRequest::builder()
                     .path(hot_reload_config_path.clone())
+                    .contents(Vec::from(self.state.config.raw_config()))
```

Comment:
So it turns out we weren't actually passing anything through to the router config, so I had to fix that up to get rid of lots of telemetry errors etc.